h/views/admin/oauthclients.py

```diff
@@ -91,7 +91,7 @@ def _template_context(self):
 
 @view_defaults(
     route_name="admin.oauthclients_edit",
-    permission="admin.oauthclients",
+    permission="admin:oauthclients",
```

Comment:
Wut... was this broken? Do you get this by virtue of the `ALL_PERMISSIONS` thing in `h.traversals.root.Root`?